test/test-runctl-patch.js

```diff
@@ -24,6 +27,7 @@ tap.test('runctl patch', function(t) {
   });
 
   t.doesNotThrow(function() {
+    // XXX(kev): Do they mean this cd? -- https://www.npmjs.com/package/cd
```

Comment:
https://github.com/strongloop/strong-supervisor/blob/master/test/helper.js#L16  <--- I regret the day I ever learned about shelljs :-(. What can I say, it seemed a good idea at the time.